roles/common/tasks/endpoint_tests.yml

```diff
@@ -6,9 +6,11 @@
   changed_when: false
   register: output
 
-- name: Check that endpoint exist {{ common_endpoint_test_id }}
+- name: |
+    TEST Check that {{ item[0] }} endpoint exists
```

Comment:
The name for this needs further updating, since a service (item[0]) can have multiple interfaces associated with it, as is the case with the endpoint tests defined in  https://github.com/infrawatch/feature-verification-tests/pull/220 (previously https://github.com/infrawatch/feature-verification-tests/pull/113)